cloudinit/sources/DataSourceIBMCloud.py

```diff
@@ -186,22 +187,30 @@ def network_config(self):
         return self._network_config
 
 
-def _read_system_uuid():
+def _read_system_uuid() -> Optional[str]:
+    """
+    Read the system uuid.
+
+    @return: the system uuid or None if not available.
```

Comment:
We decided to standarized on the [Sphix format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format), which should use `:return:`